src/pages/Home/Components/Header.js

```diff
@@ -26,7 +17,7 @@ const useStyles = makeStyles((theme) => ({
 
 const Header = () => {
   const classes = useStyles();
-  const [auth, setAuth] = React.useState(true);
+  const [auth] = React.useState(true);
```

Comment:
this was the only actual change in this file, and the fact that its autoformatted on pre commit is why everything else is autoformatted by prettier

Reply:
yup!